src/page-modules/assistant/server/journey-planner/journey-gql/via-trip.gql

```diff
@@ -0,0 +1,116 @@
+query ViaTrips(
+  $from: Location!
+  $to: Location!
+  $when: DateTime
+  $via: [ViaLocationInput!]!
+) {
+  viaTrip(
+    from: $from
+    to: $to
+    searchWindow: "PT2H" # 9 hours
+    dateTime: $when
+    via: $via
+  ) {
+    ...viaTrip
+  }
+}
+
+fragment viaTrip on ViaTrip {
+  routingErrors {
+    ...routingError
+  }
+  tripPatternsPerSegment {
+    ...tripPatternsPerSegment
+  }
+  tripPatternCombinations {
+    from
+    to
+  }
+}
+
+fragment routingError on RoutingError {
+  code
+  description
+  inputField
+}
+
+fragment tripPatternsPerSegment on ViaTripPatternSegment {
+  tripPatterns {
+    expectedStartTime
+    expectedEndTime
+    streetDistance
+    legs {
+      mode
+      distance
+      duration
+      aimedStartTime
+      expectedEndTime
+      expectedStartTime
+      realtime
+      line {
+        publicCode
+        flexibleLineType
+        notices {
+          ...notice
+        }
+      }
+      fromEstimatedCall {
+        notices {
+          ...notice
+        }
+      }
+      situations {
+        ...situation
+      }
+      fromPlace {
+        name
+        quay {
+          publicCode
+          name
+          id
+          situations {
+            ...situation
+          }
+        }
+      }
+      serviceJourney {
+        id
+        notices {
+          ...notice
+        }
+        journeyPattern {
+          notices {
+            ...notice
+          }
+        }
+      }
+      transportSubmode
+    }
+  }
+}
+
+fragment notice on Notice {
+  id
+  text
+}
+
+fragment situation on PtSituationElement {
```

Comment:
It looks like these fragments are the same as the ones defined in trip.gql and can probably be removed

Reply:
Removed👍